examples/docker-compose-rlcraft.yml

```diff
@@ -0,0 +1,20 @@
+version: "3.8"
+
+services:
+    mc:
+      image: itzg/minecraft-server:1.9.0
```

Comment:
I would prefer not to pin this to a specific version. Instead reference one of the tags.